referral-form-data-process/lib/sendDataToAPI.ts

```diff
@@ -0,0 +1,192 @@
+import axios from "axios";
+
+interface FormDataAnswersObject {
+  referrerFirstName: string | undefined;
+  referrerLastName: string | undefined;
+  requestedSupport: string | undefined;
+  clientOneFirstName: string | undefined;
+  clientOneLastName: string | undefined;
+  clientTwoFirstName: string | undefined;
+  clientTwoLastName: string | undefined;
+  clientThreeFirstName: string | undefined;
+  clientThreeLastName: string | undefined;
+  clientFourFirstName: string | undefined;
+  clientFourLastName: string | undefined;
+  clientFiveFirstName: string | undefined;
+  clientFiveLastName: string | undefined;
+  clientSixFirstName: string | undefined;
+  clientSixLastName: string | undefined;
+  clientSevenFirstName: string | undefined;
+  clientSevenLastName: string | undefined;
+  clientEightFirstName: string | undefined;
+  clientEightLastName: string | undefined;
+}
+
+export const sendDataToAPI = async (
+  formData: Record<string, string[]>,
+  documentURL: string
+) => {
+  const ENDPOINT_API = process.env.ENDPOINT_API as string;
+  const AWS_KEY = process.env.AWS_KEY as string;
+  const headers = { "x-api-key": AWS_KEY };
+  let formDataAnswersObject: FormDataAnswersObject = {
+    referrerFirstName: undefined,
+    referrerLastName: undefined,
+    requestedSupport: undefined,
+    clientOneFirstName: undefined,
+    clientOneLastName: undefined,
+    clientTwoFirstName: undefined,
+    clientTwoLastName: undefined,
+    clientThreeFirstName: undefined,
+    clientThreeLastName: undefined,
+    clientFourFirstName: undefined,
+    clientFourLastName: undefined,
+    clientFiveFirstName: undefined,
+    clientFiveLastName: undefined,
+    clientSixFirstName: undefined,
+    clientSixLastName: undefined,
+    clientSevenFirstName: undefined,
+    clientSevenLastName: undefined,
+    clientEightFirstName: undefined,
+    clientEightLastName: undefined,
+  };
```

Comment:
this could be const right?

We don't reassign the object we just update what properties it has/their value.